### PR TITLE
Update to 0.9

### PR DESCRIPTION
**Pull request by cgilmour**
Minor changes to bump the release and install new components when installing prebuilt binaries.

### CODE REVIEW

File: romana-install/roles/romana/install-agent/tasks/policy_agent.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  become_user: root
+  copy: src="agent_proxy_add_policy.sh" dest="{{ romana_bin_dir }}/" mode=0755
+  with_items:
+    - agent
+
+- name: Install Agent Proxy Delete Policy
+  become: true
+  become_user: root
+  copy: src="agent_proxy_delete_policy.sh" dest="{{ romana_bin_dir }}/" mode=0755
+
```
Comment (pritesh) [inline romana-install/roles/romana/install-agent/tasks/policy_agent.yml]: extra space

**Comment by pritesh** [comment]
rest looks good.

**Comment by pritesh** [comment]
LGTM

**Comment by jbrendel** [comment]
Ok

**Pull request merged**